srcpkgs/mold/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'mold'
+pkgname=mold
+version=1.1
+revision=1
+archs="x86_64*"
```

Comment:
Why limit the archs?

Reply:
Because I haven't yet tested it on another architectures.

Reply:
Try testing it on other archs. -Q option is xbps-src script. The Manual.md file in the repo will welp.

Reply:
Yeah, I will do that and update template accordingly.

Reply:
```
void-packages on  mold [!] ❯ ./xbps-src -a aarch64 -Q -j4 pkg mold
=> xbps-src: updating repositories for host (x86_64)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating repositories for target (aarch64)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/aarch64/aarch64-repodata' ...
aarch64-repodata: 1591KB [avg rate: 282KB/s]
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/aarch64/nonfree/aarch64-repodata' ...
aarch64-repodata: 4962B [avg rate: 61MB/s]
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/aarch64/debug/aarch64-repodata' ...
aarch64-repodata: 713KB [avg rate: 268KB/s]
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> mold-1.1_1: removing autodeps, please wait...
=> mold-1.1_1: removing autocrossdeps, please wait...
=> ERROR: mold-1.1_1: this package cannot be built for aarch64.
void-packages on  mold [!] took 15s ❯ ./xbps-src -a mips pkg mold
ERROR: XBPS_CROSS_TRIPLET is not defined!
```

```
void-packages on  mold [!] ❯ ./xbps-src -a mips-musl pkg mold
=> xbps-src: updating repositories for host (x86_64)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating repositories for target (mips-musl)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/mips-musl-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/musl/mips-musl-repodata': Not Found
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/nonfree/mips-musl-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/musl/nonfree/mips-musl-repodata': Not Found
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/debug/mips-musl-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/musl/debug/mips-musl-repodata': Not Found
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> mold-1.1_1: removing autodeps, please wait...
=> mold-1.1_1: removing autocrossdeps, please wait...
=> ERROR: mold-1.1_1: this package cannot be built for mips-musl.
```

```
void-packages on  mold [!] took 7s ❯ ./xbps-src -a mipshf-musl pkg mold
=> xbps-src: updating repositories for host (x86_64)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating repositories for target (mipshf-musl)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/mipshf-musl-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/musl/mipshf-musl-repodata': Not Found
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/nonfree/mipshf-musl-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/musl/nonfree/mipshf-musl-repodata': Not Found
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/debug/mipshf-musl-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/musl/debug/mipshf-musl-repodata': Not Found
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> mold-1.1_1: removing autodeps, please wait...
=> mold-1.1_1: removing autocrossdeps, please wait...
=> ERROR: mold-1.1_1: this package cannot be built for mipshf-musl.
```

```
void-packages on  mold [!] took 7s ❯ ./xbps-src -a ppc pkg mold
=> xbps-src: updating repositories for host (x86_64)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating repositories for target (ppc)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/ppc-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/ppc-repodata': Not Found
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/ppc-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/nonfree/ppc-repodata': Not Found
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/ppc-repodata' ...
ERROR: [reposync] failed to fetch file `https://alpha.de.repo.voidlinux.org/current/debug/ppc-repodata': Not Found
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> mold-1.1_1: removing autodeps, please wait...
=> mold-1.1_1: removing autocrossdeps, please wait...
=> ERROR: mold-1.1_1: this package cannot be built for ppc.
```

```
void-packages on  mold [!] took 7s ❯ ./xbps-src -a i686 pkg mold
=> xbps-src: updating repositories for host (x86_64)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating repositories for target (i686)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/i686-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/i686-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/i686-repodata' ...
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> mold-1.1_1: removing autodeps, please wait...
=> mold-1.1_1: removing autocrossdeps, please wait...
=> ERROR: mold-1.1_1: this package cannot be built for i686.
```

I think, it can only build for `x86_64`

Reply:
"this package cannot be built for" is a message one see when `archs=` is set. Remove it before building for other archs.